packages/server/src/api/controllers/application.ts

```diff
@@ -197,6 +199,30 @@ export async function fetch(ctx: UserCtx) {
     }
   }
 
+  // Get all builder sessions in each app
+  const sessions = await builderSocket?.getRoomSessions(appIds)
```

Comment:
Ideally I would break this out into an SDK function, would be useful to have a `sdk.users.sessions.list(appIds)` - that way we wouldn't pollute the application controller as much!

Reply:
The only external call is the builder socket function, the rest is just local logic for enriching apps. `sdk.users.sessions.list(appIds)` would just be a straight proxy through to `builderSocket?.getRoomSessions(appIds)` which then wouldn't reduce the implementation size at all, unless we moved some of the local logic into it as well (i.e. generating the map) which would pull out a few extra lines, but then it's not really a `list` function... Any ideas?

Reply:
Changed the function name, I think it would be better to do `const appsWithSessions = sdk.users.sessions.enrichApps(apps)`

Reply:
Sounds good!